### ISSUE TITLE
Building in ARM or aarch64 failed because unable to load seccomp

### ISSUE THREAD

**Issue by lucernae** [lucernae/devcontainer-nix#26]
Error shown when running the Dockerfile build (either using buildx or build)

```shell
docker buildx build --platform linux/arm64/v8 .
```

```
error: unable to load seccomp BPF program: Invalid argument
```

**Comment by lucernae** [comment]
I have no idea what happened, but found this issue: https://github.com/NixOS/nix/issues/5258
Seems that the BPF filter failed.
I'm currently running the build inside Apple Silicon -> rancher desktop -> lima vm/moby -> docker buildx -> Dockerfile statement

Need to test if the build only failed in this condition.

**Comment by lucernae** [comment]
Fixed by including `filter-syscalls = false` in the build pipeline.
Fixed in [v1.1.0](https://github.com/lucernae/devcontainer-nix/releases/tag/v1.1.0)
